src/gitlab/functions/e2e-start-pipeline-button.ts

```diff
@@ -1,32 +1,48 @@
 import { isFrontend } from './is-frontend';
+import { isApp } from './is-app';
 import { GitlabPipelines } from '../services/gitlab-api/pipelines-api';
 import { GitlabJobs } from '../services/gitlab-api/jobs-api';
 import { createHashString } from './hash-variables';
 
+const FrontendImageJob = {
+    name: 'exponea docker image',
+    imageRegex: /digest: (sha256:[^ ]*) size: \d+/gm,
+    imageUrlParam: 'fe_version'
+}
+
+const AppImageJob = {
+    name: 'prod image',
+    imageRegex : /^Built Docker image\:.*digest:.*(sha256\:.+)$/m,
+    imageUrlParam: 'be_version'
+};
+
 export function createRunE2eButton(mergeRequestId: number) {
-    if (!isFrontend()) {
+    if (!isFrontend() && !isApp()) {
         return Promise.resolve();
     }
 
     if (mergeRequestId == null) {
         return Promise.resolve();
     }
 
+    const imageJob = isFrontend() ? FrontendImageJob : AppImageJob;
```

Comment:
Maybe creating a hash  with from `FrontendImageJob` and `AppImageJob`? You could then use key `window.monar_GLOBALS.project` as a key in such hash and get values using the key

Reply:
Ok, I did it with project id because that is sure to not change